google-chart.html

```diff
@@ -384,6 +404,7 @@
             var chart = chartAndData[0];
             var data = chartAndData[1];
             google.visualization.events.addOneTimeListener(chart, 'ready', function() {
+              this._setImageUri(this.imageURI);
```

Comment:
Instead of unconditionally setting this on load, my suggestion is to cache it on first access. I believe this should work fine if someone setups on a binding on the property:

```
(function()) {
Polymer({
  properties: {
    _imageUri: {type: String, value: null}
  },
  get imageURI() {
    if (!this._imageUri) {
      this._imageUri = this._chartObject.getImageURI();
     }
    return this._imageUri;
  }
});

))();